distribution/src/config/jvm.options

```diff
@@ -72,10 +72,10 @@
 
 # specify an alternative path for heap dumps; ensure the directory exists and
 # has sufficient space
-${heap.dump.path}
+@heap.dump.path@
 
 # specify an alternative path for JVM fatal error logs
-${error.file}
+@error.file@
 
 ## GC logging
 -Xlog:gc*,gc+age=trace,safepoint:file=${loggc}:utctime,pid,tags:filecount=32,filesize=64m
```

Comment:
Missed one. Need to update `${loggc}` here.

Reply:
f3fb120ef100c343f02069a64b2d3a7e27f346b1